source/common/network/connection_impl.cc

```diff
@@ -753,6 +756,27 @@ void ConnectionImpl::flushWriteBuffer() {
   }
 }
 
+std::ostream& operator<<(std::ostream& os, Connection::State connection_state) {
```

Comment:
Implementing this operator in the middle of this cc file is a bit odd.  Consider moving to an anonymous namespace at the top of the file or to the library that defines the Connection::State enum class.

Reply:
Moved it to anonymous namespace at the top of this file. Done.